docs/release-notes/8.2.0-1.md

```diff
@@ -0,0 +1,66 @@
+# Percona Server for MySQL 8.2.0-1 (2024-01-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.2.0-1] includes all the features and bug fixes available in the
+MySQL 8.2 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+This release merges the MySQL 8.2 code base. In this release Percona does not introduce any new features, improvements or bug fixes.
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.2 and included in Percona Server for MySQL are the following:
+
+* The insert operations caused `records_in_range` to read too many disk blocks.
+
+* System schema tables with INSTANT ADD columns created before 8.0.29 are incompatible with MySQL versions higher than 8.0.29. Performing DMLs on these tables causes an unexpected server exit.
+
+* A FTS parser plugin now handles single character tokens correctly.
+
+### Deprecation or removal
+
+The following items are deprecated in this release and using any of these items may cause a warning:
```

Comment:
Maybe

A future release may remove deprecated variables and options. Using these deprecated items may cause a warning. We recommend migrating from deprecated variables and options as soon as possible.

This release deprecates the following variables and options:
			

			* The `binlog_transaction_dependency_tracking` server system variable.
			

			* The `old` and `new` server system variables.
			

			* The `--character-set-client-handshake` server variable.
			

			* `INFORMATION_SCHEMA.PROCESSLIST` and the `SHOW PROCESSLIST` command.
			

			* The `performance_schema_show_processlist` variable.

Reply:
updated